integration/simulation/devnetwork/node.go

```diff
@@ -205,7 +214,8 @@ func NewInMemNodeOperator(operatorIdx int, config ObscuroConfig, nodeType common
 		l1Client:     l1Client,
 		l1Wallet:     l1Wallet,
 		logger:       logger,
-		sqliteDBPath: dbFile,
+		sqliteDBPath: sqliteDBPath,
```

Comment:
this looks super confusing having both sqlite and leveldb.

If my understanding is correct, one is the db path for the enclave, and one of the host.
Shouldn't we name them like this then?